ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -464,6 +464,12 @@ def __array__(self, dtype=None, /):
         if not dtype:
             return self
         return np_frontend.array(self, dtype=dtype)
+    
+    def __array_wrap__(self, out_arr, context=None):
```

Comment:
We need to specify the end of positional args here
and also to align with the numpy documentation they specified the first argument name to be `array`
```suggestion
    def __array_wrap__(self, out_arr, context=None, /):
```

Reply:
done but after doing this I'm getting an error which indicates that the shape of the returned array from the __array_wrap__ function does not match the expected shape. I tried using `return np_frontend.array(array).reshape(self.shape)` but getting the same error

Reply:
We are already there going to explore this will get back to you😅